packages/material-ui-lab/src/ToggleButton/ToggleButton.test.js

```diff
@@ -54,10 +54,10 @@ describe('<ToggleButton />', () => {
       </ToggleButton>,
     );
 
-    const root = getByTestId('root');
-    expect(root).to.have.class(classes.root);
-    expect(root).to.have.class(classes.sizeSmall);
-    expect(root).not.to.have.class(classes.sizeLarge);
+    const rootNode = getByTestId('root');
```

Comment:
It's not a node. It's more specifically an Element. This is important because the class matcher wouldn't make sense on a node.